dmutils/content_loader.py

```diff
@@ -402,7 +402,10 @@ def _get_single_question_data(self, form_data):
                 "assurance": form_data.get(self.id + '--assurance'),
             }
 
-        return {self.id: value if value else None}
```

Comment:
I missed the commit where this was added.
Should we be doing the same thing for checkboxes (line 384)? This makes it possible to delete subsections by unchecking values though.